common-lib/src/test/scala/com/gu/mediaservice/lib/metadata/ImageMetadataConverterTest.scala

```diff
@@ -419,5 +420,37 @@ class ImageMetadataConverterTest extends FunSpec with Matchers {
     imageMetadata.peopleInImage should be (Set("person 1","person 2"))
   }
 
+  it("should cope with awful dates") {
```

Comment:
Could this have a name that explains a little more about what functionality this is actually testing? 🙏

Reply:
I thought that was quite explicit!

Reply:
I since updated the PR description (without asking consent). Is that enough or should it be in code?

Reply:
"Awful" _is_ both descriptive and explicit, but it doesn't really say anything about what we consider awful, or what specifically about all of the dates we're asserting against is awful. Is this "should accept incomplete dates" or "should accept dates in a non-standard format" or both or something else? I could probably infer that from looking at all of the assertions, or by looking up the PR, but it would also be useful to have that context right there in the test itself.